scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXJoystick")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\Joystick\Scenes\JoystickExample.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXMRKeyboard")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\MixedRealityKeyboard\Scenes\MixedRealityKeyboardExample.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXNNKeyboard")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\NonNativeKeyboard\Scenes\NonNativeKeyboardExample.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXPulseShader")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\PulseShader\Scenes\PulseShaderExamples.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXRiggedHandVisual")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\RiggedHand\Scenes\RiggedHandVisualizer.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXScrollObject")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\ScrollingObjectCollection\Scenes\ScrollingObjectCollection.unity"
+   }
+   ElseIf ("${{ parameters.Scene }}" -eq "EXSolvers")
+   {
+       $sceneList += "Assets\MRTK\Examples\Experimental\Solvers\DirectionalIndicatorExample.unity"
+   }
+
+   $extraArgs = ''
+   If ("${{ parameters.Platform }}" -eq "UWP")
+   {
+       $extraArgs += '-buildTarget WSAPlayer -buildAppx'
+   }
+   ElseIf ("${{ parameters.Platform }}" -eq "Standalone")
+   {
+       $extraArgs += "-buildTarget StandaloneWindows"
+   }
+   If ("${{ parameters.UnityArgs }}" -ne "none")
+   {
+       $extraArgs += " ${{ parameters.UnityArgs }}"
+   }
+   If ("${{ parameters.ScriptingBackend }}" -eq ".NET")
+   {
+       $extraArgs += " -scriptingBackend 2"
+   }
+
+   $proc = ''
+   If ("${{ parameters.Scene }}" -eq "LeapMotion")
+   {
+       Write-Output 'Coming soon importing Leap Motion core Unity package. This just builds an audio example scene.'
+       Start-Process -FilePath "$editor" -ArgumentList "-projectPath $projectPath -executeMethod Microsoft.MixedReality.Toolkit.Build.Editor.UnityPlayerBuildTools.StartCommandLineBuild -sceneList Assets\MRTK\Examples\Demos\Audio\Scenes\AudioLoFiEffectDemo.unity -logFile $($logFile.FullName) -batchMode -${{ parameters.Arch }} -buildOutput $outDir $extraArgs -CacheServerIPAddress $(Unity.CacheServer.Address) -logDirectory $logDirectory" -PassThru
+   }
+   Else
+   {
+       Start-Process -FilePath "$editor" -ArgumentList "-projectPath $projectPath -executeMethod Microsoft.MixedReality.Toolkit.Build.Editor.UnityPlayerBuildTools.StartCommandLineBuild -sceneList $sceneList -logFile $($logFile.FullName) -batchMode -${{ parameters.Arch }} -buildOutput $outDir $extraArgs -CacheServerIPAddress $(Unity.CacheServer.Address) -logDirectory $logDirectory" -PassThru
+   }
+
+       $ljob = Start-Job -ScriptBlock { param($log) Get-Content "$log" -Wait } -ArgumentList $logFile.FullName
```

Comment:
nit: odd indentation

Reply:
I fixed the indentation issue for the $ljob variable.